pkgs/by-name/ip/iproute2/package.nix

```
@@ -55,8 +55,9 @@ stdenv.mkDerivation rec {
   '';
 
   outputs = [
-    "out"
     "dev"
+    "out"
```

```suggestion
    "out"
    "dev"
```
because $out should remain the default

---

thx done